<?xml version="1.0"?>
<launch>
  <param name="map_name"  value="map6"/>
  <param name="inicio_x"  value="2"/>
  <param name="inicio_y"  value="2"/>
  
  <!-- Servicios -->
  <node pkg="planificacion_pkg" name="lector_mapa_csv" type="lector_mapa_csv"/>
  <node pkg="planificacion_pkg" name="detectar_esquinas" type="detectar_esquinas"/>
  <node pkg="planificacion_pkg" name="servicio_obstaculos" type="servicio_obstaculos"/>
  
  <!-- Planificador -->
  <node pkg="planificacion_pkg" name="planificador_grafo_dijkstra" type="planificador_grafo_dijkstra" output="screen" launch-prefix="xterm -e"/>

</launch>
